src/components/Footer.ts

```diff
@@ -22,19 +25,55 @@ export class Footer extends LitElement {
         color: var(--bkd-func-fg-black);
       }
 
+      footer {
+        display: flex;
+        justify-content: space-between;
+      }
+
       .copyright {
         font-size: 0.8125rem;
         font-weight: 300;
         letter-spacing: 0.02rem;
         word-spacing: 0.05rem;
       }
 
+      .footer-nav a {
+        font-size: 14px;
```

Comment:
Grundsätzlich würde ich Schriftgrössen und alles was mit der Schrift wachsen/schrumpfen müsste in `rem` angeben. Also hier 14/16 = `0.875rem`.